crates/sui-types/src/temporary_store.rs

```diff
@@ -271,6 +275,10 @@ impl<S> TemporaryStore<S> {
             written,
             deleted,
             events: TransactionEvents { data: self.events },
+            max_binary_format_version: ProtocolConfig::get_for_version(
+                self.protocol_config.version,
+            )
+            .move_binary_format_version(),
```

Comment:
Surely this is just:
```suggestion
            max_binary_format_version: self.protocol_config.move_binary_format_version(),
```

Reply:
That was dumb...